### ISSUE TITLE
Cannot use other/modified config.json files

### ISSUE THREAD

**Issue by dudebrains01** [BlinkTagInc/gtfs-to-html#68]
I am having a lot of difficulty with GTFS-to-HTML, starting with the fact that as soon as I use another config file (that I created by copying the config-sample and renaming then modifying) or I save the original config-sample, I get a syntax error
Here is the command prompt output I get ( would use the code feature but the presence of apostrophies in the output makes it imposible, sorry)
Error: Cannot find configuration file at `c:\users\brody\appdata\roaming\npm\node_modules\gtfs-to-html\config-sample.json`. Use config-sample.json as a starting point, pass --configPath option
    at C:\Users\brody\AppData\Roaming\npm\node_modules\gtfs-to-html\bin\gtfs-to-html.js:56:19

Error: SyntaxError: Unexpected token ﻿ in JSON at position 0
SyntaxError: Unexpected token ﻿ in JSON at position 0
    at JSON.parse (<anonymous>)
    at getConfig (C:\Users\brody\AppData\Roaming\npm\node_modules\gtfs-to-html\bin\gtfs-to-html.js:41:23)

**Comment by brendannee** [comment]
Can you share with me the `config.json` file you are using?

**Comment by dudebrains01** [comment]
Here are the files I am using.
Please excuse the delay.

**Comment by brendannee** [comment]
No files were linked to that post - you can link me to it or just paste the contents of config.json

**Comment by dudebrains01** [comment]
Sorry about that
Here is the contents:

{
  "mongoUrl": "mongodb://localhost:27017/gtfs",
  "agencies": [
    {
      "agency_key": "STO",
      "url": "http://www.contenu.sto.ca/GTFS/GTFS.zip"
    }
  ],
  "beautify": false,
  "coordinatePrecision": 5,
  "dateFormat": "D MMM YYYY",
  "daysShortStrings": ["Lun", "Mar", "Mer", "Jeu", "Ven", "Sam", "Dim"],
  "daysStrings": ["Lundi", "Mardi", "Mercredi", "Jeudi", "Vendredi",
"Samedi", "Dimanche"],
  "defaultOrientation": "vertical",
  "effectiveDate": "26 Aout 2019",
  "interpolatedStopSymbol": "•",
  "linkStopUrls": false,
  "mapboxAccessToken":
"sk.eyJ1IjoiYnVzZHVkZSIsImEiOiJjanlxOHB2MWkxbm10M2hzODA0M20yMjhlIn0.11nIrggB_MVtY0Ogzo2s7A",
  "menuType": "simple",
  "noDropoffSymbol": "‡",
  "noHead": false,
  "noPickupSymbol": "**",
  "noServiceSymbol": "—",
  "requestDropoffSymbol": "†",
  "requestPickupSymbol": "***",
  "showArrivalOnDifference": 0.2,
  "showMap": true,
  "showOnlyTimepoint": true,
  "showRouteTitle": true,
  "showStopCity": false,
  "showStopDescription": false,
  "skipImport": false,
  "sortingAlgorithm": "common",
  "timeFormat": "h:mma",
  "verbose": true,
  "zipOutput": false
}

**Comment by brendannee** [comment]
That config.json you sent is valid and worked for me - here is the exact config.json file zipped (unzip it and give it a try)

[config.json.zip](https://github.com/BlinkTagInc/gtfs-to-html/files/3494967/config.json.zip)

I did just push an updated version of GTFS-to-HTML 0.17.9 which will give better error messages when trying to parse the config file so it may give you more insight on what is wrong. 

I did notice that the GTFS file you are using does not have a calendar.txt file and relies on calendar_dates.txt to specify specific dates that are served. This means that you'll need to create a timetables.txt file in your GTFS to use GTFS-to-HTML which will specify exactly which service_ids and days of the week you want to generate timetables for. https://github.com/BlinkTagInc/gtfs-to-html#build-timetablestxt
